### PR TITLE
gh #126 Updating source yaml file for display

### PR DESCRIPTION
**Pull request by drissya-k**
Updating the source yaml file for display as parsing error occured during testing.
For source devices, if source ports are zero and non of the tests runs for source device except init and term. So added 1 port in yaml and type as HDMI for source devices.

### CODE REVIEW

File: profiles/source/Source_4K_Display.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 dsDisplay:
   Type: source
   Name: xione
+  Number_of_ports: 0
+  Video_Ports: []
```
Comment (bhanucbp) [inline profiles/source/Source_4K_Display.yaml:5]: why do we need Aspect Ratio here? Interface document already defines the default value returned if device in not connected.

Comment (drissya-k) [reply profiles/source/Source_4K_Display.yaml:5]: done the changes

File: profiles/source/Source_4K_Display.yaml [hunk 2]
```diff
@@ -1,9 +1,7 @@
 dsDisplay:
   Type: source
   Name: xione
+  Number_of_ports: 0
```
Comment (bhanucbp) [inline profiles/source/Source_4K_Display.yaml:4]: There is a problem here for source devices, if source ports are zero and non of the tests runs for source device except init and term. Its better to add 1 port in yaml and type should be HDMI for source devices

Comment (drissya-k) [reply profiles/source/Source_4K_Display.yaml]: done

**Review by thanushreevani — APPROVED** [review]

**Review by bhanucbp — APPROVED** [review]
ship it

**Pull request merged**